iot-hub/Samples/device/DeviceStreamingSample/DeviceStreamSample.cs

```diff
@@ -21,48 +21,44 @@ public DeviceStreamSample(DeviceClient deviceClient)
+                else
+                {
+                    await _deviceClient
+                        .AcceptDeviceStreamRequestAsync(streamRequest, cts.Token)
+                        ;
+
+                    using ClientWebSocket webSocket = await DeviceStreamingCommon
+                        .GetStreamingClientAsync(streamRequest.Url, streamRequest.AuthorizationToken, cts.Token);
 
-                        using (ClientWebSocket webSocket = await DeviceStreamingCommon
-                            .GetStreamingClientAsync(streamRequest.Url, streamRequest.AuthorizationToken, cancellationTokenSource.Token)
-                            .ConfigureAwait(false))
-                        {
-                            WebSocketReceiveResult receiveResult = await webSocket
-                                .ReceiveAsync(new ArraySegment<byte>(buffer, 0, buffer.Length), cancellationTokenSource.Token)
-                                .ConfigureAwait(false);
-                            Console.WriteLine("Received stream data: {0}", Encoding.UTF8.GetString(buffer, 0, receiveResult.Count));
+                    WebSocketReceiveResult receiveResult = await webSocket
+                        .ReceiveAsync(new ArraySegment<byte>(buffer, 0, buffer.Length), cts.Token);
+                    Console.WriteLine("Received stream data: {0}", Encoding.UTF8.GetString(buffer, 0, receiveResult.Count));
 
-                            await webSocket
-                                .SendAsync(new ArraySegment<byte>(buffer, 0, receiveResult.Count), WebSocketMessageType.Binary, true, cancellationTokenSource.Token)
-                                .ConfigureAwait(false);
-                            Console.WriteLine("Sent stream data: {0}", Encoding.UTF8.GetString(buffer, 0, receiveResult.Count));
+                    await webSocket
+                        .SendAsync(
+                            new ArraySegment<byte>(buffer, 0, receiveResult.Count),
+                            WebSocketMessageType.Binary,
+                            true,
+                            cts.Token);
+                    Console.WriteLine("Sent stream data: {0}", Encoding.UTF8.GetString(buffer, 0, receiveResult.Count));
 
-                            await webSocket
-                                .CloseAsync(WebSocketCloseStatus.NormalClosure, string.Empty, cancellationTokenSource.Token)
-                                .ConfigureAwait(false);
-                        }
-                    }
-                    else
-                    {
-                        await _deviceClient.RejectDeviceStreamRequestAsync(streamRequest, cancellationTokenSource.Token).ConfigureAwait(false);
-                    }
+                    await webSocket
+                        .CloseAsync(WebSocketCloseStatus.NormalClosure, string.Empty, cts.Token);
                 }
 
-                await _deviceClient.CloseAsync().ConfigureAwait(false);
+                await _deviceClient.CloseAsync();
```

Comment:
@abhipsaMisra you think we should not call close on our clients in the samples?

Reply:
We usually try to initialize the device client inside a disposable block, and pass the client to the sample program.  If that is not possible, I would recommend that our sample `.dispose()` the client as a part of cleanup, rather than `.close()`.

Reply:
Dispose is called from Program.cs via the `using` keyword.

So my question is, should I remove this call?